load-tests/mysql_select/deployment/mysql_deployment.yml

```
@@ -46,7 +46,7 @@ spec:
           args:
             - "-o"
             - "/tmp/data/init.sql"
-            - "https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-mysql/main/load-tests/select/scripts/init.sql"
+            - "https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-mysql/master/load-tests/mysql_select/scripts/init.sql"
```

```suggestion
            - "https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-mysql/master/load-tests/select/scripts/init.sql"
```
Shall we remove "mysql_" prefix? As we have moved the scripts to mysql repo, IMO this is redundant

---

Fixed in 8a4f044